modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -2968,7 +2968,11 @@ definition {
 
 			ObjectAdmin.goToLayoutsTab();
 
-			WaitForElementPresent(locator1 = "ObjectAdmin#KEBAB_MENU");
```

Comment:
I removed this function because it is already being used in the selectKebabMenuOption macro just below.